common/src/request_id_prepending.rs

```diff
@@ -29,7 +29,7 @@ pub fn detach_prefix_from_request_id(request_id: &str) -> (String, String) {
         return (String::from(""), String::from(""));
     }
     let mut splitted = request_id.splitn(2, SEPARATOR);
-    let prefix = splitted.next().unwrap();
+    let prefix = splitted.next().unwrap_or_default();
```

Comment:
Maybe unwrap_or_unreachable()?

Reply:
Done, see comment above for `unwrap_or_unreachable` extension.